client/src/pages/dashboard/Dashboard.js

```diff
@@ -22,11 +22,13 @@ function Dashboard() {
     let proposal_lst = response.data;
     /* Initialize false <checked> attributes for each proposal; used for checkbox tracking
     while in deleting mode */
+    console.log("proposals fetched " + proposal_lst)
     proposal_lst.forEach(proposal => {
       proposal_lst.checked = false;
     })
 		setProposals(proposal_lst);
     setFilteredProposalsList(proposal_lst);
+    firstProposal(proposal_lst);
```

Comment:
maybe rename func to "setPreviewToFirstProposal" bc its not clear what "firstProposal" does